### ISSUE TITLE
[FEATURE REQUEST] Support latest `pino-http` (v7)

### ISSUE THREAD

**Issue by micalevisk** [iamolegga/nestjs-pino#929]
Currently we have this semver range:

https://github.com/iamolegga/nestjs-pino/blob/3b39d171fa25853d023145341b7bbafbb30f8afd/package.json#L89-L90

I'm not sure if you need to change something in the code to support `pino-http@7`

The release notes: https://github.com/pinojs/pino-http/releases/tag/v7.0.0

**Comment by hisabimbola** [comment]
It'll be good to have this released

**Comment by micalevisk** [comment]
I _think_ we can use the latest already.

In my case, I've changed `customLogLevel` signature from this:

```
customLogLevel(res: ServerResponse, err: Error)
```

to this:

```
customLogLevel(_, res: ServerResponse, err: Error)
```

because now it receives the request object as the first parameter

**Comment by micalevisk** [comment]
I believe you can support both. Then if someone report that it doesn't work with v6 anymore, you can release a new major

**Comment by iamolegga** [comment]
ok, will do it this week

**Comment by iamolegga** [comment]
published in `2.6.0`
